datumaro/plugins/ade20k2020_format.py

```diff
@@ -22,7 +22,10 @@
 
 
 class Ade20k2020Path:
-    MASK_IMAGE_PATTERN = re.compile('_seg|_parts_|instance')
+    MASK_PATTERN = re.compile(r'''\w+_seg\.\w+
+        | \w+_parts_\d+\.\w+
+        | instance_\w+\.\w+
+    ''', re.VERBOSE)
```

Comment:
Isn't `\w+` too restrictive? It won't accept digits and special characters like - and _.

Reply:
Okay, I'll add `-` and spaces, regarding digits and underscores In the [docs](https://docs.python.org/3/library/re.html) says that `\w` accepts them.